listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -410,7 +417,7 @@ export default class RecentListens extends React.Component<
                       .map((listen) => {
                         return (
                           <tr
-                            key={listen}
+                            key={listen.listened_at}
```

Comment:
let's use `listened_at-recording_msid-username` here. `listened_at` can still be duplicated, for example in follow page.